llvm/test/CodeGen/X86/pr99396.ll

```diff
@@ -0,0 +1,59 @@
+; RUN: llc < %s -mtriple=i386-unknown-freebsd -enable-misched -relocation-model=pic | FileCheck %s
+
+target datalayout = "e-m:e-p:32:32-p270:32:32-p271:32:32-p272:64:64-i128:128-f64:32:64-f80:32-n8:16:32-S128"
+target triple = "i386-unknown-freebsd15.0"
+
+@c = external local_unnamed_addr global ptr
+
+declare i32 @fn2() local_unnamed_addr
+
+declare i32 @fn3() local_unnamed_addr
+
+define noundef i32 @fn4() #0 {
+entry:
+  %0 = load i32, ptr @fn4, align 4
+; CHECK: movl fn4@GOT(%ebx), %edi
+; CHECK-NEXT: movl (%edi), %edx
+  %1 = load ptr, ptr @c, align 4
```

Comment:
Use named values in tests

Reply:
Sorry, I haven't got much experience with writing these tests, what does this mean? I'm intending here to check the exact registers used.

Reply:
%0 should be %load or some other name etc.